services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/TransferManagerConfiguration.java

```diff
@@ -97,37 +97,21 @@ public static Builder builder() {
 
     public static final class Builder {
 
-        private Boolean followSymbolicLinks;
-        private Integer maxDepth;
+        private Boolean uploadDirectoryFollowSymbolicLinks;
+        private Integer uploadDirectoryMaxDepth;
         private Executor executor;
 
 
-        public Builder followSymbolicLinks(Boolean followSymbolicLinks) {
-            this.followSymbolicLinks = followSymbolicLinks;
+        public Builder uploadDirectoryFollowSymbolicLinks(Boolean uploadDirectoryFollowSymbolicLinks) {
```

Comment:
Do we need getter/setter pairs for `uploadDirectoryFollowSymbolicLinks` and `uploadDirectoryMaxDepth`?

Reply:
Bean-style getter/setter pairs are actually only needed for builder implementations that implement _public_ builder APIs. They are not needed in this class since this is an internal class.

Reply:
Ah, that's right.